### ISSUE TITLE
The 'Featured Reef' card does not match the designs. Also, change to 'Featured Site'

### ISSUE THREAD

**Issue by drewjgray** [aqualinkorg/aqualink-app#264]
Let's use the design as intended, both copied here for reference. 

<img width="668" alt="Screen Shot 2020-09-28 at 4 17 52 PM" src="https://user-images.githubusercontent.com/6283065/94526510-51607280-01d1-11eb-8f27-11d554789cd1.png">

<img width="664" alt="Screen Shot 2020-09-28 at 4 16 15 PM" src="https://user-images.githubusercontent.com/6283065/94526493-4d345500-01d1-11eb-9f49-59867149f198.png">

**Comment by ericboucher** [comment]
@drewjgray can you be a bit more specific? We had to make a few adjustments to make the design more responsive + now we have the concept of "selected reef"

We can move things but it would be great to get an itemized lists of what you want to match in priority

**Comment by ericboucher** [comment]
To start @tleraj' let's:
- make the image larger
- move the reef name and region to be above the graph
- have the top title alternate between `Featured Site` and `Selected Site`

**Comment by drewjgray** [comment]
@ericboucher @tleraj The above is perfect. 

Let's also move the button to be below the Heat Stress value. It's a vertical alignment so I think it will work well for mobile too.

**Comment by ericboucher** [comment]
@drewjgray this whole area doesn't work very well on mobile as is, unfortunately. Which is why we moved the button out in the first place actually. The way we made the UI work on mobile was by moving the metrics columns to a metrics row on mobile.

<img width="383" alt="Screen Shot 2020-09-30 at 12 28 47 AM" src="https://user-images.githubusercontent.com/16843267/94623216-f784a900-02b3-11eb-820d-88a695cdb3ce.png">


We can decide to move the button for desktop but make the designs more different on mobile maybe? @ewolfer67 thoughts?
